src/app/(dashboard)/page.tsx

```
@@ -13,5 +16,6 @@ export default async function Home() {
     redirect("/sign-in");
   }
 
+  return <p>{greeting}</p>;
   return <HomeView />;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)